packages/nimble-components/src/rich-text/mention-list-box/index.ts

```diff
@@ -0,0 +1,309 @@
+import { Notifier, Observable, observable } from '@microsoft/fast-element';
```

Comment:
The biggest architectural concern we have is how much custom code exists here. Ideally we'd share a lot more of it with other components like the select and combobox. Is this handwritten? Copied from FAST? Copied from other Nimble components?

Reply:
As we updated the architecture to inherit from the list box, closing this thread. Please let us know if you have any concerns further.